core-data/resolvers.js

```diff
@@ -33,15 +38,19 @@ export async function* getAuthors() {
 }
 
 /**
- * Requests a entity's record from the REST API.
+ * Requests an entity's record from the REST API.
  *
  * @param {Object} state  State tree
  * @param {string} kind   Entity kind.
  * @param {string} name   Entity name.
  * @param {number} key    Record's key
  */
 export async function* getEntityRecord( state, kind, name, key ) {
-	const entity = getEntity( kind, name );
+	const entities = yield* getKindEntities( state, kind );
```

Comment:
I've reached the extent of my knowledge of asynchronous generators 😄 Is an `await` keyword needed on this line if `getKindEntities` may only return (resolve) after an async `apiRequest` ?